manifests/pipecd/grafana-dashboards/control-plane/incoming-request.json

```diff
@@ -506,47 +589,15 @@
   "style": "dark",
   "tags": [],
   "templating": {
-    "list": [
-      {
-        "allValue": null,
-        "current": {
-          "selected": false,
-          "text": "server-api",
-          "value": "server-api"
-        },
-        "datasource": "Prometheus",
-        "definition": "label_values(envoy_cluster_name)",
-        "description": null,
-        "error": null,
-        "hide": 0,
-        "includeAll": false,
-        "label": null,
-        "multi": false,
-        "name": "api",
-        "options": [],
-        "query": {
-          "query": "label_values(envoy_cluster_name)",
-          "refId": "StandardVariableQuery"
-        },
-        "refresh": 1,
-        "regex": "",
-        "skipUrlSync": false,
-        "sort": 0,
-        "tagValuesQuery": "",
-        "tags": [],
-        "tagsQuery": "",
-        "type": "query",
-        "useTags": false
-      }
-    ]
+    "list": []
   },
   "time": {
     "from": "now-6h",
     "to": "now"
   },
   "timepicker": {},
   "timezone": "",
-  "title": "Incoming Request",
-  "uid": "3jeba3R7z",
-  "version": 1
+  "title": "Prometheus",
```

Comment:
The file name is "incoming-request" but the title is "Prometheus".

Reply:
Oops. OMG